MIGRATION.md

```diff
@@ -2,6 +2,20 @@
 
 This document provides information about breaking changes and their migrations for major versions of Kirby Design System.
 
+## Version 8
+
+- [Version 8](#version-8)
+  - [Components](#components-v8)
+    - [List](#list-v8)
+
+<h3 id="components-v8">Components</h3>
+
+<h4 id="list-v8">List</h4>
```

Comment:
What is the reason for using html here instead of markdown (`<h3>` vs `###`)?

Reply:
It's because of the links in the table of contents for each version. It was working correctly while we only had a table of contents for version 6, where we fx. created a link like this `[Components](#components)`. This part `#components` automatically creates a link to a heading with the title "components". But that becomes a problem when we have multiple table of contents in the file, as the link will only point to the first instance of the title. Fx. `[Components](#components)` in the table of contents for version 8 will point to the "components" title in version 6. By using HTML we are able to make links using id's instead.

There is a markdown solution that also works, but I think the HTML solution is preferable. To achieve the same effect in markdown, then we would need to create the links by adding the index of the title as a suffix like this: [Components](#components-0). But that becomes messy if there is a title that is not included in a table of contents. Fx:

```
- [Version 8](#version-8)
  - [Platform Support](#platform-support-1)
  - [Components](#components-2)
    - [Alert](#alert-1)
    - [List](#list-2)
    - [Page](#page-0)

- [Version 7](#version-7)
  - [Components](#components-1)
    - [List](#list-1)

- [Version 6](#version-6)
  - [Platform Support](#platform-support-0)
  - [Components](#components-0)
    - [Alert](#alert-0)
    - [List](#list-0)
```

It starts to become tricky in version 8 in the example above, because some titles have already appeared multiple times and some appears for the first time, which results in a different index, so we have to keep track of how many times a title appears to make a link, instead of just using an HTML tag and link to the specified id.

That was a long response  😂

Reply:
That makes a lot of sense. Would a markdown solution where we just append the version work?
```
- [Version 6](#version-6) 
   - [Platform Support](#platform-support-6) 
   - [Components](#components-6)
```

Or will there still be instances where that becomes messy? Otherwise I think it is the most straight forward approach that makes sense for whoever will have to do it next time.

Reply:
I'll approve and you can just use your best judgement on what you like the most.

Reply:
That could work, but it then we would have to include the number in the actual title like this:

```
- [Version 6](#version-6) 
   - [Platform Support](#platform-support-6) 
   - [Components](#components-6)

Platform support 6
Lorem ipsum......

Components 6
Lorem ipsum.....
```
I think the output of the HTML solution looks cleaner :)